src/transformers/models/bloom/tokenization_bloom_fast.py

```diff
@@ -32,38 +32,22 @@
 
 logger = logging.get_logger(__name__)
 
-VOCAB_FILES_NAMES = {"vocab_file": "vocab.json", "merges_file": "merges.txt", "tokenizer_file": "tokenizer.json"}
+VOCAB_FILES_NAMES = {"tokenizer_file": "tokenizer.json"}
 
 PRETRAINED_VOCAB_FILES_MAP = {
-    "vocab_file": {
-        "bloom": "https://huggingface.co/bloom/resolve/main/vocab.json",
-        "bloom-medium": "https://huggingface.co/bloom-medium/resolve/main/vocab.json",
-        "bloom-large": "https://huggingface.co/bloom-large/resolve/main/vocab.json",
-        "bloom-xl": "https://huggingface.co/bloom-xl/resolve/main/vocab.json",
-        "distilbloom": "https://huggingface.co/distilbloom/resolve/main/vocab.json",
-    },
-    "merges_file": {
-        "bloom": "https://huggingface.co/bloom/resolve/main/merges.txt",
-        "bloom-medium": "https://huggingface.co/bloom-medium/resolve/main/merges.txt",
-        "bloom-large": "https://huggingface.co/bloom-large/resolve/main/merges.txt",
-        "bloom-xl": "https://huggingface.co/bloom-xl/resolve/main/merges.txt",
-        "distilbloom": "https://huggingface.co/distilbloom/resolve/main/merges.txt",
-    },
     "tokenizer_file": {
-        "bloom": "https://huggingface.co/bloom/resolve/main/tokenizer.json",
-        "bloom-medium": "https://huggingface.co/bloom-medium/resolve/main/tokenizer.json",
-        "bloom-large": "https://huggingface.co/bloom-large/resolve/main/tokenizer.json",
-        "bloom-xl": "https://huggingface.co/bloom-xl/resolve/main/tokenizer.json",
-        "distilbloom": "https://huggingface.co/distilbloom/resolve/main/tokenizer.json",
+        "bloom": "https://huggingface.co/bigscience-catalogue-data-dev/byte-level-bpe-tokenizer-no-norm-250k-whitespace-and-eos-regex-alpha-v3-dedup-lines-articles/resolve/main/tokenizer.json",
```

Comment:
You should point to https://huggingface.co/bigscience/tokenizer it's the version that doesn't require you to have to build your own `tokenizers`. You just need to bump it >1.12.0

Reply:
fixed in afa817db234907d7b691504b11bce74e616331b0 ! but I think the repo of the tokenizer is private?